### PR TITLE
Add Slippage to Swap & Updated Wallet Styling

### LINKED ISSUE #39: Slippage for Swap

### PR DESCRIPTION
**Pull request by eupston**
## Description

updated swap component with slippage functionality 
closes #39 

#### Changes

- add slippage functionality swap
- updated pool and swap with global connect wallet btn
- updated wallet button to match styling spec

### CODE REVIEW

**Comment by ken-centrality** [comment]
@eupston FYI, I think this has conflict due to my merge of #44 and #49. Let me know if you want me to resolve it

**Review by ken-centrality — CHANGES_REQUESTED** [review]
Looks goods, 2 minor things:

- [x] Slippage input should have a `min` as 1% (!?) and possibly a `max` attribute 100%
- [x] In the PR message, the list of changes is kept within the `>` quote block, can you remove it and make it a nice tidy list using `-`

**Review by aidan-starke — CHANGES_REQUESTED** [review]
Looks good!
Couple things:
 - I know the design has the wallet displaying the user's account address, but I prefer seeing the username there, what do we reckon @ken-centrality ?
 - Slippage text is only referring to `CPAY` - needs to update with whatever token is being exchanged 
 - Can you pass in poolColors from Pool so that `WITHDRAW FROM POOL` button is the purple colour?

**Comment by ken-centrality** [comment]
@aidan-starke you mean the account name eh? then yeah I agree, it's friendlier than a rand string of characters

**Comment by aidan-starke** [comment]
> @aidan-starke you mean the account name eh? then yeah I agree, it's friendlier than a rand string of characters

yup!

File: components/Wallet.tsx [hunk 1]
```diff
@@ -63,15 +63,11 @@ const Wallet: React.FC<{}> = () => {
...
 					}}
 				>
 					{isWalletConnected
-						? selectedAccount?.address.slice(0, 9) +
-						  "..." +
-						  selectedAccount?.address.slice(
-								selectedAccount?.address.length - 4,
-								selectedAccount?.address.length
-						  )
+						? selectedAccount?.meta?.name.substring(0, 17)
```
Comment (ken-centrality) [inline components/Wallet.tsx:70]: for future case, you can use the CSS rule `text-overflow: ellipsis` to truncate and add `...` to the text when it's overflow

**Review by ken-centrality — APPROVED** [review]

File: components/Wallet.tsx (hunk 1 above)
Comment (eupston) [reply components/Wallet.tsx:70]: O awesome, I'll change that in this PR

**Review by aidan-starke — APPROVED** [review]

**Pull request merged**